bundles/org.openhab.binding.comfoair/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1465 @@
+	</channel-type>
+
+	<channel-type id="analog3_volt" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Analog Input 3 Voltage Level</label>
+		<description>Voltage level of analog input 3</description>
+		<category>Number</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="analog3_min" advanced="true">
+		<item-type>Number</item-type>
+		<label>Analog Input 3 Min</label>
+		<description>Minimum setting for analog input 3</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="analog3_max" advanced="true">
+		<item-type>Number</item-type>
+		<label>Analog Input 3 Max</label>
+		<description>Maximum setting for analog input 3</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="analog3_value" advanced="true">
+		<item-type>Number</item-type>
+		<label>Analog Input 3 Target</label>
+		<description>Target setting for analog input 3</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="analog4_volt" advanced="true">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Analog Input 4 Voltage Level</label>
+		<description>Voltage level of analog input 4</description>
+		<category>Number</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="analog4_min" advanced="true">
+		<item-type>Number</item-type>
+		<label>Analog Input 4 Min</label>
+		<description>Minimum setting for analog input 4</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="analog4_max" advanced="true">
+		<item-type>Number</item-type>
+		<label>Analog Input 4 Max</label>
+		<description>Maximum setting for analog input 4</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="analog4_value" advanced="true">
+		<item-type>Number</item-type>
+		<label>Analog Input 4 Target</label>
+		<description>Target setting for analog input 4</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="RF_min" advanced="true">
+		<item-type>Number</item-type>
+		<label>RF Input Min</label>
+		<description>Minimum setting for RF input</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="RF_max" advanced="true">
+		<item-type>Number</item-type>
+		<label>RF Input Max</label>
+		<description>Maximum setting for RF input</description>
+		<category>Number</category>
+		<state min="0" max="100" step="1" readOnly="false" />
+	</channel-type>
+
+	<channel-type id="RF_value" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
You should consider changing this to `Number:Dimensionless` so that you can use Percent units.
She same applies to any other channel uses percentages.

Reply:
I would rather not do this as these are not indicated as percentage values in the device documentation. Addtionally I rechecked the limits for these channels. Found out that the maximum for the ´min_value´s is defined with 99, and AFAIK this can not be defined for UoM type channels. So I would rather leave these as is.

Reply:
You can leave as is then.